patches/api/0194-Add-Player-Client-Options-API.patch

```diff
@@ -127,6 +130,19 @@ index 0000000000000000000000000000000000000000..f7f171c4ee0b8339b2f8fbe82442d65f
 +        this.chatColors = chatColors;
 +        this.skinparts = skinParts;
 +        this.mainHand = mainHand;
++        this.allowsServerListings = false;
++    }
++
++    public PlayerClientOptionsChangeEvent(@NotNull Player player, @NotNull Map<ClientOption<?>, ?> options) {
```

Comment:
Add an ApiStatus.Internal annotation to the ctor. no plugin should really need to construct this event I think.